src/js/common/components/FAQBody.jsx

```diff
@@ -49,22 +50,31 @@ export default class FAQBody extends Component {
         <Suspense fallback={<></>}>
           <OpenExternalWebSite
             linkIdAttribute="weVoteEducationWebsite"
-            url="http://WeVoteEducation.org"
+            url="https://www.WeVoteEducation.org"
```

Comment:
Unfortunately WeVoteEducation.org is not SSL yet either, and won't be in the near future.

Reply:
It is, try it in a browser  https://www.WeVoteEducation.org